src/Database/PostgreSQL/PQTypes/Model/Trigger.hs

```diff
@@ -35,6 +38,38 @@ import Data.Text qualified as Text
 import Database.PostgreSQL.PQTypes
 import Database.PostgreSQL.PQTypes.SQL.Builder
 
+-- | Timing for a regault trigger.
+--
+-- @since 1.17.0.0
+data TriggerRegularTiming
+  = -- | An @AFTER@ trigger.
```

Comment:
Can you use `-- ^` style for constructors here and below? IMO it reads better and it's consistent with how fields in other data types are described.

I'm surprised fourmolu didn't reformat this :thinking:

Reply:
I tried, but we might have an option in the fourmolu config. I'll take a look.

Reply:
I didn't find a way to use Haddock comments with `-- ^`, I don't know why, they are always re-written with `-- |`.